docs/HowTo/UpdatePatroni.md

```diff
@@ -0,0 +1,129 @@
+
+We recommend moving from 1.6.5 to 2.0.1. If your team wishes to upgrade further, please feel free to do so.
+
+## Step 1: Preparation
+
+Before you begin, as is always best practice, please make sure that you back up your database. 
+
+You may also wish to switch the leader of your Patroni cluster over to pod 0 in your stateful set. This shouldn't be necessary, but will probably make things easier on you.
+
+This list of steps also requires that the stateful set not automatically redeploy on image change. Please ensure that this is correct before proceeding.
+
+Lastly make sure that the current state of your Patroni cluster is healthy, that all members are communicating properly, that you have plenty of space in your PVCs, and that there is a low `Lag in MB` value when you check `patronictl list` to see the status of your cluster. You can run this command from the terminal of any pod in your cluster.
+
+## Step 2: Put your Patroni cluster into Maintenance Mode
+
+Go to the terminal of any pod in your patroni cluster and run the following command:
+
+`patronictl pause`
+
+This should produce the following message:
+
+`Success: cluster management is paused`
+
+You can also double-check your cluster's mode using the following command:
+
+`patronictl list`
+
+If your cluster is in maintenance mode, you will something like the following:
+
+```
+    + Cluster: patroni-001 (7065071675363201060) ------+----+-----------+
+    | Member        | Host         | Role    | State   | TL | Lag in MB |
+    +---------------+--------------+---------+---------+----+-----------+
+    | patroni-001-0 | 10.97.19.146 | Replica | running |  2 |         0 |
+    | patroni-001-1 | 10.97.13.94  | Leader  | running |  2 |           |
+    +---------------+--------------+---------+---------+----+-----------+
+    Maintenance mode: on
+ ```
+
+If your cluster is not in maintenance mode, the bottom line will be missing.
+
+## Step 3: Build a New Image
+
+**Note: If you are still building from the dockerfile in the BCDevops/platform-services repo, please note that this file is no longer being maintained.** It does contain this particular fix, in order to allow teams using that file to perform this upgrade without having to deal with the additional complication of switching Dockerfiles. For what it's worth, I switched between the files several times during testing and never saw an issue - however, it would be understandable if this seems like an unnecessary risk. In either case, please note that you should change your buildconfig, whether now or in the future, to point to the dockerfile in bcgov/patroni-postgres-container in take advantage of future file maintenance.
```

Comment:
also mention that the patroni image from `bcgov` namespace will not be maintained either

Reply:
will it not? i thought we were continuing to keep that one updated.

Reply:
hmmm i'm not sure then. I don't see a buildconfig that output to local imagestream in bcgov namespace, only to docker and artifactory now.